packages/utils/src/lib/__snapshots__/report-to-md.spec.ts.snap

```diff
@@ -15,11 +15,11 @@ _Audits: 52_
 
 |Category|Score|Audits|
 |:--:|:--:|:--:|
-|Performance|1|2/1|
-|Bug prevention|19|16/19|
-|Code style|13|13/13|
+|Performance|0.9215000000000001|2/1|
```

Comment:
Yes! :tada:

We should format the scores, by the way, but we can leave that to #148